app/models/listing.rb

```diff
@@ -18,7 +18,7 @@ class Listing < ActiveRecord::Base
   accepts_nested_attributes_for :listing_images
 
   # def self.max_guests
-  #   where("max_guests >= ?", max_guests)
+  #   where("max_guests >= ?", max_guests)  #this class method is equivilant to the max_guest scope above
```

Comment:
Line is too long. [105/80]<br>Incorrect indentation detected (column 2 instead of 6).